.env

```diff
@@ -25,7 +25,14 @@ API_INTERNAL_PORT=8081
 API_INTERNAL_TEST_PORT=8082
 
 # Brapi Server Variables
-BRAPI_CORE_URL=https://test-server.brapi.org/
-BRAPI_PHENO_URL=https://test-server.brapi.org/
-BRAPI_GENO_URL=https://test-server.brapi.org/
-BRAPI_REFERENCE_SOURCE=breeding-insight.org
\ No newline at end of file
+BRAPI_CORE_URL=http://localhost:8083/
```

Comment:
should this be pre-defined here as localhost:8083 if it depends on the brapi configuration below? Should it use the BRAPI_SERVER_PORT variable instead?

Reply:
Yeah I like that idea!

Reply:
I tried to do `http://localhost:${BRAPI_SERVER_PORT}` for this, but I got micronaut errors because of it. I think we may have to leave it has localhost:8083 for now and revisit it later if it causes too many issues.